declarative-examples/simple-examples/stepsandWrappers.groovy

```diff
@@ -0,0 +1,19 @@
+pipeline {
```

Comment:
`stepsandWrappers.groovy` looks weird -- what are step sand wrappers? -- `stepsAndWrappers.groovy` would be better, because unambiguous.